Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -43,6 +46,129 @@ public class AttackByEventArgs : EventArgs
         public IEntity AttackWith { get; set; }
     }
 
+    #region Tools
+
+    /// <summary>
+    /// This interface gives components behavior when being clicked on or "attacked" by a user with a tool in their hand
+    /// </summary>
+    public interface IToolAct
+    {
+        /// <summary>
+        /// Called when using a wrench on an entity
+        /// </summary>
+        bool ToolAct(ToolActEventArgs eventArgs) => false;
```

Comment:
This method isn't used, is it?

Reply:
Ah, I forgot to actually call it! I fixed that now